care.js

```diff
@@ -311,18 +311,25 @@ function catSay(text) {
 
 function getAnsiArt(textToSay) {
   var artFileRegex = /.ansi$/;
+  var say = config.say;
+
+  // Choose random parrot every time
+  if (say === 'random') {
+    var available = ['parrot', 'bunny', 'llama', 'cat', 'yeoman', 'mario', 'ironman', 'minions', 'panda'];
+    say = available[Math.floor(Math.random()*available.length)];
+  }
 
   // If config.say is custom art file path, then return custom art
-  if (artFileRegex.test(config.say)) {
-    return ansiArt.get({ filePath: config.say, speechText: textToSay });
+  if (artFileRegex.test(say)) {
```

Comment:
But this will never be true, right? You already overwritten it to be one of the 9 hardcoded options